script/scriptgen.py

```diff
@@ -674,7 +674,7 @@ def gen_read_files(self, f):
         if self.assets and self.assets_flavor:
             for k, v in self.asset_folders.items():
                 self.p(
-                    """rsync -4 --list-only $rsync_pwd_option PRODUCTPATH/{}/*{} | awk '{{ $1=$2=$3=$4=""; print substr($0,5); }}' >> __envsub/files_asset.lst""".format(
+                    """rsync -4 --list-only $rsync_pwd_option PRODUCTPATH/{}/*{} | awk '{{ $1=$2=$3=$4=""; print substr($0,5); }}' >> __envsub/files_asset.lst || true""".format(
```

Comment:
Discussed in jitsi. This may be the wrong part of the generator, see https://github.com/os-autoinst/openqa-trigger-from-obs/pull/238#issuecomment-1886916332 - I will look into that after coming up with a fake rsync that produces successful or error output, and see how grep and awk respond to that.

Reply:
So it seems the `read_files_repo` value in `cfg.py:23` is the basis for that rsync call and the `p` function in `scriptgen.py` actually replaces the target file that is being piped into i.e. `__Envsub/files_repo.lst`